### ISSUE TITLE
compass部署问题

### ISSUE THREAD

**Issue by Pandas886** [cubefs/compass#2]
compass启动前是否一定要将Hadoop、Spark、Canal、MySQL、Kafka、Redis、Zookeeper、Elasticsearch全部都启动好，
然后配置好application-hadoop.yml和 compass_env.sh，执行./bin/start_all.sh即可？

**Comment by nilnon** [comment]
Yes, please refer to the deployment guide for more details. [deployment document](https://github.com/cubefs/compass/blob/main/document/manual/deployment.md)
